packages/storybook-html/src/components/core/date.story.mdx

```diff
@@ -28,18 +28,19 @@ These date formats are designed for
     <tbody>
         <tr>
           <td>
-          10:00 AM <br />
-          2:30 PM
+          10:00AM <br />
```

Comment:
a.m/p.m preferred as standard
Space also needed 10:00 a.m instead of 10:00am
Please check below link
https://writingexplained.org/am-or-a-m-pm-or-p-m-do-i-capitalize-am-and-pm#:~:text=The%20first%20and%20most%20common,daily%20at%2010%3A05%20a.m.

Reply:
even if we use AM/PM space is needed from what i read on the link above